src/g1.rs

```diff
@@ -418,6 +417,25 @@ impl G1Affine {
     }
 }
 
+/// A nontrivial third root of unity in Fp
+pub const BETA: Fp = Fp([
```

Comment:
Can we use `Fp::from_raw_unchecked` here instead so that the constants listed here represent the actual field element versus its Montgomery form?

Reply:
I'm not 100% sure what you mean here, I changed it out but since `from_raw_unchecked` just constructs `Fp(v)` directly, the array passed to it is exactly the same.